lib/files/schema/simulationparameter.json

```diff
@@ -0,0 +1,313 @@
+{
+    "title": "SimulationParameter",
+    "description": "It is the complete set of EnergyPlus Simulation Settings.",
+    "type": "object",
+    "properties": {
+      "type": {
+        "title": "Type",
+        "enum": [
+          "SimulationParameter"
+        ]
+      },
+      "output": {
+        "$ref": "#/definitions/SimulationOutput"
+      },
+      "run_period": {
+        "$ref": "#/definitions/RunPeriod"
+      },
+      "timestep": {
+        "title": "Timestep",
+        "default": 6,
+        "minimum": 1,
+        "type": "integer"
+      },
+      "simulation_control": {
+        "$ref": "#/definitions/SimulationControl"
+      },
+      "shadow_calculation": {
+        "$ref": "#/definitions/ShadowCalculation"
+      },
+      "sizing_parameter": {
+        "$ref": "#/definitions/SizingParameter"
+      }
+    },
+    "required": [
+      "type"
+    ],
+    "definitions": {
+      "SimulationOutput": {
+        "title": "SimulationOutput",
+        "description": "Lists the desired outputs from the simulation and the format in which to report them.",
+        "type": "object",
+        "properties": {
+          "type": {
+            "title": "Type",
+            "enum": [
+              "SimulationOutput"
+            ]
+          },
+          "reporting_frequency": {
+            "title": "Reporting_Frequency",
+            "default": "Hourly",
+            "enum": [
+              "Timestep",
+              "Hourly",
+              "Daily",
+              "Monthly",
+              "Annual"
+            ],
+            "type": "string"
+          },
+          "include_sqlite": {
+            "title": "Include_Sqlite",
+            "default": true,
+            "type": "boolean"
+          },
+          "include_html": {
+            "title": "Include_Html",
+            "default": false,
+            "type": "boolean"
```

Comment:
Currently there is no way not to include the html, and sqlite reports.

Reply:
That's a bit of a shame because it forces the generation of potentially heavy files that might not be used by the simulator. This might be something we can live with, though.

Reply:
FYI, @tanushree04 , Julien addressed this with an edit to the IDF ForwardTranslator:
https://github.com/NREL/OpenStudio/issues/3786#issuecomment-558318610

So we should be able to support this capability but it's not something that you need to worry about here in the energy-model-measure since the ForwardTranslator is not a part of the OpenStudio model object. Instead, it's something that we will address in the IDF translation process and I have some ideas for how to make use of this from Grasshopper if I find I can't do it through OpenStudio CLI.

Also, judging from the code that Julien pointed me towards, I think the exposure of SummaryReports is also going to be something that gets addressed in the ForwardTranslator object and not as a part of the energy-model-measure.